config/config.go

```diff
@@ -111,6 +111,7 @@ type AGConfig struct {
 	AgreementBatchSize           uint64           // The number of nodes that the agbot will process in a batch.
 	FullRescanS                  uint64           // The number of seconds between policy scans when there have been no changes reported by the exchange.
 	MaxExchangeChanges           int              // The maximum number of exchange changes to request on a given call the exchange /changes API.
+	MaxContractPrelaunchTimeM    int64            // The maximum numbers of minutes to wait for workload to start in an agreement
```

Comment:
This is not an agbot config. Please move it to the edge config (struct Config) instead. And please change the it to "MaxAgreementPrelaunchTimeM"

Reply:
Sure. Renamed and moved to `Config`